### PR TITLE
Added Some Tricks

### PR DESCRIPTION
**Pull request by samuelbeek**
Set maximum amount of charachters in UITextView
Add person to addresbook (which was extremely hard to figure out)
Use hex color codes

### CODE REVIEW

File: cheat_sheet.swift [hunk 1]
```diff
@@ -118,10 +120,95 @@ UIApplication.sharedApplication().idleTimerDisabled = true
 // 11) dismiss the keyboard when pressing return in a UITextView
 
 func textView(textView: UITextView!, shouldChangeTextInRange: NSRange, replacementText: NSString!) {
-  if (replacementText == "\n") {
-    myTextView.resignFirstResponder()
-  }
+    if (replacementText == "\n") {
```
Comment (zackshapiro) [inline cheat_sheet.swift]: Thanks

File: cheat_sheet.swift [hunk 2]
```diff
@@ -118,10 +120,95 @@ UIApplication.sharedApplication().idleTimerDisabled = true
...
+    }
 }
 
 
-// more coming soon
+// 12) set a maximum amount of characters to a UITextView
+// add UITextViewDelegate to your viewcontroller like this:
+// class EditViewController : UIViewController, UITextViewDelegate {
+
+func textView(textView: UITextView, shouldChangeTextInRange range: NSRange, replacementText text: String) -> Bool {
```
Comment (zackshapiro) [inline cheat_sheet.swift]: I wonder if there's a way to make this function a little more obvious about what it does. Would be cool to either

A) Rename it to be more semantic about what it does
B) Feed in a character limit Int

Just some thoughts, nothing preventing me from merging

File: cheat_sheet.swift [hunk 3]
```diff
@@ -118,10 +120,95 @@ UIApplication.sharedApplication().idleTimerDisabled = true
...
+                var success:Bool = false
+                
+                //Updated to work in Xcode 6.1
+                var error: Unmanaged<CFErrorRef>? = nil
+                //Updated to error to &error so the code builds in Xcode 6.1
+                success = ABRecordSetValue(newContact, kABPersonFirstNameProperty, firstName, &error)
+                success = ABRecordSetValue(newContact, kABPersonLastNameProperty, lastName, &error)
+                success = ABRecordSetValue(newContact, kABPersonJobTitleProperty, jobTitle, &error)
+                
+                if(phoneNumber != nil) {
```
Comment (zackshapiro) [inline cheat_sheet.swift]: add space after `if` please to keep with styling throughout doc

File: cheat_sheet.swift [hunk 4]
```diff
@@ -118,10 +120,95 @@ UIApplication.sharedApplication().idleTimerDisabled = true
...
+            red: CGFloat((rgb & 0xFF0000) >> 16) / 255.0,
+            green: CGFloat((rgb & 0x00FF00) >> 8) / 255.0,
+            blue: CGFloat(rgb & 0x0000FF) / 255.0,
+            alpha: CGFloat(1.0)
+        )
+    }
+    
+}
+
+// more comming soon
```
Comment (zackshapiro) [inline cheat_sheet.swift]: Will you add items to the readme too please?

This is awesome, Samuel! Thanks for contributing!

**Comment by zackshapiro** [comment]
@samuelbeek Ready to merge in pending these small changes

File: cheat_sheet.swift (hunk 4 above)
Comment (samuelbeek) [inline cheat_sheet.swift]: Hi Zack, 

I changed the readme.md too; made some code a bit clearer (the maxLength one was definitly unreadible for those who don't code in swift very often) :+1:

Have a good day :)

**Comment by zackshapiro** [comment]
Thanks Sam! Keep 'em coming

**Pull request merged**